docs/plugins/conversion_table.py

```diff
@@ -0,0 +1,117 @@
+from dataclasses import dataclass
+from decimal import Decimal
+from typing import Any, Literal
+
+from pydantic_core import CoreSchema, core_schema
+
+
+@dataclass
+class Row:
+    field_type: type[Any]
+    input_type: type[Any]
+    mode: Literal['lax', 'strict', 'both']
+    input_format: Literal['python', 'JSON', 'python, JSON']
+    condition: str | None = None
+    valid_examples: list[Any] | None = None
+    invalid_examples: list[Any] | None = None
+    core_schema: type[CoreSchema] = None
+
+
+table_infos: list[Row] = [
+    Row(
+        str,
+        str,
+        'both',
+        'python, JSON',
+        core_schema=core_schema.StringSchema,
+    ),
+    Row(
+        str,
+        bytes,
+        'lax',
+        'python',
+        condition='assumes UTF-8, error on unicode decoding error',
+        valid_examples=[b's'],
+        invalid_examples=[b'\x81'],
+        core_schema=core_schema.StringSchema,
+    ),
+    Row(
+        str,
+        bytearray,
+        'lax',
+        'python',
+        condition='assumes UTF-8, error on unicode decoding error',
+        valid_examples=[bytearray(b's' * 5)],
+        invalid_examples=[bytearray(b'\x81' * 5)],
+        core_schema=core_schema.StringSchema,
+    ),
+    Row(
+        bytes,
+        bytes,
+        'both',
+        'python',
+        core_schema=core_schema.BytesSchema,
+    ),
+    Row(
+        bytes,
+        str,
+        'lax',
+        'python, JSON',
+        valid_examples=['foo'],
+        core_schema=core_schema.BytesSchema,
+    ),
+    Row(
+        bytes,
+        bytearray,
+        'lax',
+        'python',
+        valid_examples=[bytearray(b's' * 5)],
+        core_schema=core_schema.BytesSchema,
+    ),
+    Row(
+        int,
+        int,
+        'both',
+        'python, JSON',
+        condition='max abs value 2^64 - i64 is used internally',
+        core_schema=core_schema.IntSchema,
```

Comment:
can you add an invalid example.

Reply:
I've added an invalid example. Also, I've separated the `strict` and `lax` mode for `int` type in table because it raises `ValidationError` for number > limit in `strict` mode but it will limit the input to the max in `lax` mode.
I've added descriptions for these in the condition field but I am not sure about the wording.

I've added tests for both above-mentioned cases.